hack/jenkins/e2e.sh

```diff
@@ -53,6 +54,7 @@ else
       export MASTER_SIZE=${MASTER_SIZE:-"t2.small"}
     else
       export MASTER_SIZE=${MASTER_SIZE:-"g1-small"}
```

Comment:
Our performance goals are phrased in the context of an n1-standard-4 master (https://github.com/GoogleCloudPlatform/kubernetes/issues/6454)